src/ansible_creator/resources/common/patterns/sample_pattern/playbooks/site.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Weather forecast
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Ensure the airport code has been defined
+      ansible.builtin.assert:
+        that: "{{ airport_code is defined }}"
+        fail_msg: Define the airport code ('-e airport_code=RDU')
+
+    - name: Clone the latitude longtude repo locally
+      ansible.scm.git_retrieve:
```

Comment:
Since we're using both ansible.scm and community.general here but not adding them to the collections requirements (which is good), Can we show the user a warning message noting "The sample pattern relies on the ansible.scm and community.general collections. Please manually add these to the collection's requirements file to use the sample pattern as-is."